features/polyExplorer/src/screens/entityDetails/entityDetails.jsx

```diff
@@ -19,17 +19,17 @@ const EntityDetails = () => {
     const [swiper, setSwiper] = useState(null);
 
     const cityImageMap = {
-        München: "munich",
-        "Mountain View": "mountainview",
-        Wiesbaden: "wiesbaden",
-        Berlin: "berlin",
-        Dubai: "dubai",
-        Luxembourg: "luxembourg",
-        "WILMINGTON, New Castle": "wilmington",
-        "WILMINGTON, DELAWARE": "wilmington",
-        WILMINGTON: "wilmington",
-        Cupertino: "cupertino",
-        DUBLIN: "dublin",
+        münchen: "munich",
```

Comment:
But in many cases you're mapping a name to itself... Is this necessary now that everything is in lowercase?

Reply:
The mapped ones have a picture with a location. So to know whether a company has a picture we look in the map. Lowercase was necessary now that companies have WILMINGTON and Wilmington as their location